src/lambdas/getProductById.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+const dbHelper = require("../helpers/db.helper");
+const { responseHeaders } = require("../common/config");
+const { StatusCodes } = require("http-status-codes");
+const validate = require("uuid-validate");
+
+const UUID_VERSION = 4;
+
+const getProductByIdQuery = (id) => `
+  select products.id, products.title, products.description, products.price, products.image, stocks."count" from products 
+  left join stocks
+  on stocks.product_id = products.id
+  where products.id = '${id}'
```

Comment:
@OlgaGnatenko, this part of the code is vulnerable to `SQL injection` attacks. I would suggest that you change using `Parameterized query`

Here is an example of using parameterized query
```
const query = {
  text: 'INSERT INTO users(name, email) VALUES($1, $2)',
  values: ['brianc', 'brian.m.carlson@gmail.com'],
}
```
More info can be found here: https://node-postgres.com/features/queries